tests/components/test_input_select.py

```diff
@@ -91,39 +77,48 @@ def test_config_options(self):
         ]
 
         self.assertTrue(input_select.setup(self.hass, {
```

Comment:
Also all these tests have to go through `bootstrap._setup_component` because validators can change the config object (ie via `cv.ensure_list`)